tests/test_credits.py

```diff
@@ -1,21 +1,41 @@
-import pytest
-import os
-from credit import load_credits, logger
+import pytest, os, json, logging
+from credit import load_credits, Credits
 from unittest import mock
 
 
+logger = logging.getLogger('main.py')
```

Comment:
You probably want to import the actual `logger` from `main.py`

Reply:
You said that here do without importing too.

Reply:
Sorry for that, importing a `logger` in tests is ok as we are never going to use it to log anything, only to check that it was used correctly.